templates/smartstart/guide/hardware-setup.md

```diff
@@ -8,15 +8,15 @@ context:
+Board certification for Ubuntu is a prerequisite for commercialisation with SMART START. Canonical has [certified a selection x86 and ARM-based](https://certification.ubuntu.com/iot) off-the-shelf boards for Ubuntu. Customers can choose one, and benefit from the shortest time to market (two weeks delivery). Alternatively, customers can purchase [Canonical’s device enablement](/smartstart/guide/device-enablement) service as an add-on to get any board on their choice certified for Ubuntu.
 
 ## Certified boards
 
 Boards certified for Ubuntu span x86, ARM, 32bit and 64bit architectures. These boards are generally available in development versions and industrial-grade versions. Certified boards are listed below.
 
-| amd64|i386|arm64|armhf|
-| --- | --- | --- | --- |
-|[Intel NUC7CJYH](https://certification.ubuntu.com/hardware/201805-26256)<br> [Intel NUC7PJYH](https://certification.ubuntu.com/hardware/201805-26252)<br> [Intel NUC7i3DNHE](https://certification.ubuntu.com/hardware/201802-26085)<br> [Intel NUC7i5DNHE](https://certification.ubuntu.com/hardware/201802-26086)<br> [Intel NUC7i5DNHE](https://certification.ubuntu.com/hardware/201802-26087)<br> [Intel TANK-870-Q170](https://certification.ubuntu.com/hardware/201808-26450) | Intel Edison  | Qualcomm Dragonboard 410c<br> Raspberry Pi model 3<br> Raspberry Pi model 4| Raspberry Pi model 2<br>Raspberry Pi model 3<br>Raspberry Pi model 4<br>Raspberry Pi CM3|
+| amd64                                                                                                                                                                                                                                                                                                                                                                                                                                                                                | i386         | arm64                                                                       | armhf                                                                                    |
+| ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ------------ | --------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------- |
```

Comment:
Do you need all of this?

Reply:
Oh, it is a markdown table prettier thing, obviously it isn't helping in this case, but quiet useful most of the time. I assume you want me to uninstall it from my code editor and remove all the space?